network/epcommon/endpoint_common.go

```diff
@@ -189,7 +189,7 @@ func BlockIPAddresses(bridgeName string, action string) error {
 func EnableIPForwarding(ifName string) error {
 	// Enable ip forwading on linux vm.
 	// sysctl -w net.ipv4.ip_forward=1
-	cmd := fmt.Sprintf(enableIPForwardCmd)
+	cmd := fmt.Sprint(enableIPForwardCmd)
```

Comment:
why do we change this? what's the use?

Reply:
we are not doing a string interpolation so it's unnecessary to call `Sprintf`, and using the minimal `Sprint` is more correct